src/app/tasks/task-item/task-item.component.ts

```diff
@@ -15,7 +15,7 @@ export class TaskItemComponent {
 
   @Output() taskDeletedEvent = new EventEmitter<void>();
 
-  @Input() taskItem: Task = emptyTask;
+  @Input() taskItem: BasicTask = {} as BasicTask;
```

Comment:
again why are we assigning an empty object to a variable

Reply:
I couldn't find proper typescript documentation about initializing objects. I used Way 4 in this answer [https://stackoverflow.com/a/52616197](url).
1. I didn't want `@Input() taskItem: Task | null = null;` because then I would have to check for null every time the input is referenced
2. To me initializing taskItem with empty object is the same as creating a constant that contains useless values.
Should I reverse these changes?

Reply:
Ok, that's fine